app/views/sequence/show_sequence.html.erb

```diff
@@ -36,18 +36,26 @@
           <td class="bold"><%= :LOCUS.t %>:</td>
           <td><%= @sequence.locus %></td>
         </tr>
-        <tr>
-          <td class="bold align-top"><%= :BASES.t %>:</td>
-          <td class="col-md-9 monospace max-width-text">
-            <%= @sequence.bases %></td>
-        </tr>
-        <tr>
-          <td class="bold"><%= :DEPOSIT.t %>:</td>
-          <td>
-            <% if @sequence.deposit? %>
-              <%= sequence_archive_link(@sequence) %>:
-              <%= sequence_accession_link(@sequence) %>
-            <% end %>
+        <% if @sequence.bases.present? %>
+          <tr>
+            <td class="bold align-top"><%= :BASES.t %>:</td>
+            <td class="col-md-9 monospace max-width-text">
+              <%= @sequence.bases %></td>
+          </tr>
+        <% end %>
+        <% if @sequence.deposit? %>
+          <tr>
+            <td class="bold"><%= :DEPOSIT.t %>:</td>
+            <td>
+                <%= sequence_archive_link(@sequence) %>:
+                <%= sequence_accession_link(@sequence) %>
+            </td>
+          </tr>
+        <% end %>
+        <tr align="center">
```

Comment:
I'm not in love with the way this looks in show_sequence. 
Any suggestions?
How about a button like this: 
![blastbuttondown](https://user-images.githubusercontent.com/45460/34342540-1417b142-e968-11e7-9b04-f4a50026ce01.jpg)
And how do I implement a button outside of a form without using js?